frontend/src/metabase/dashboard/components/DashboardHeader.styled.tsx

```diff
@@ -149,13 +149,6 @@ export const HeaderButtonsContainer = styled.div<TypeForItemsThatRespondToNavBar
     padding-right: 1rem;
     border-top: 1px solid ${color("border")};
   }
-
-  ${Button.Root} {
-    padding: 0.5rem 0.75rem;
-    &:hover {
-      color: ${color("brand")};
-    }
-  }
```

Comment:
Moved to `frontend/src/metabase/dashboard/containers/DashboardHeader.styled.jsx` so we have style that target the same elements in the same file.